packages/apps/reputation-oracle/server/src/common/pipes/validation.ts

```diff
@@ -24,4 +23,31 @@ export class HttpValidationPipe extends ValidationPipe {
       ...options,
     });
   }
+
+  private formatErrorsSnakeCase(
```

Comment:
Can we use `[mapChildrenToValidationErrors](https://github.com/nestjs/nest/blob/9bff3c98e49a256b37cbb520676cd0deec444487/packages/common/pipes/validation.pipe.ts#L291C13-L291C42)` to flatten errors and handle only case changes on our end?
It would help to avoid potential bugs if something changes in nest and also make code more readable